src/components/pageTemplates/articlePage.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "/styles/variables.scss";
+
+.newsArticle {
+  max-width: 50rem;
+
+  figure {
+    margin-left: 0;
+  }
+}
+
+.pageDivider {
+  background-color: #000;
+  content: "";
+  display: block;
+  height: 2px;
+  margin-top: 32px;
+  width: 64px;
```

Comment:
Use HDS spacing values for margin and width: https://hds.hel.fi/design-tokens/spacing